package.json

```diff
@@ -88,7 +88,7 @@
     "fuse.js": "^6.5.3",
     "gulp": "^4.0.2",
     "hammerjs": "2.0.7",
-    "hugo-bin": "^0.68.0",
+    "hugo-bin-extended": "^0.98.0",
```

Comment:
@gumaerc Is this updating the hugo version we use in prod? Or just locally in this repo?